packages/icons/src/svg/facebook-2.svg

```diff
@@ -1,3 +1,4 @@
 <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I would suggest other name for those coloured social icons, because `facebook-2` isn't descriptive.